features/mobile/not_safe_for_work.feature

```diff
@@ -0,0 +1,74 @@
+@javascript @mobile
+Feature: Not safe for work
+  Background:
+    Given following users exist:
+      | username    | email             |
+      | pr0n king   | tommy@pr0n.xxx    |
```

Comment:
When you are changing/writing any tests, please replace all domains with example.com. We've linked to some really stupid sites in the past and I don't wanna do that...